.github/workflows/build-publish.yml

```diff
@@ -0,0 +1,79 @@
+name: 'Build Publish Chainlink'
+
+on:
+  push:
+    branches:
+      - 'release/**'
+
+jobs:
+
+  build-publish-chainlink:
+    # buildjet-2vcpu-ubuntu-1804
+    runs-on: ubuntu-1804
+    steps:
+      - uses: actions/checkout@v2
+      - name: 'Install AWS CLI'
+        run: |
+          if [[ -z "$BUILDER" ]]; then
+            exit 0
+          fi
+
+          curl -sSL "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+          unzip awscliv2.zip
+          sudo ./aws/install
+          rm awscliv2.zip
+          aws --version
+
+      - name: 'Docker login for ECR using temporary creds'
```

Comment:
Docker has an action for this, see usage here: https://github.com/docker/login-action#aws-elastic-container-registry-ecr